package.json

```diff
@@ -42,6 +42,7 @@
 		"package": "vsce package ."
 	},
 	"dependencies": {
+		"@sap/consume-services-types": "^0.3.6",
```

Comment:
The toolkit should not be dependant on the consume services. It is lower level extension.
You should either define the IService in the toolkit, or use some more "abstract type" (preferable not any)